Makefile

```diff
@@ -3,7 +3,7 @@
 PACKAGES = $(shell go list ./... | grep -v /vendor/)
 
 ifneq ($(shell uname), Darwin)
-	EXTLDFLAGS = -extldflags "-static" $(null)
+	EXTLDFLAGS = -extldflags "-static-libgcc -static-libstdc++" $(null)
 else
```

Comment:
@tboerger yea, I think it is best to try to build with just `-static` on a platform that doesn't have known issues. For example instead of building on ubuntu one can build on alpine or another base image. This is an example 0.4 yaml that builds on arm:
https://github.com/armdrone/drone/blob/master/.drone.yml